deploy/cluster_role.yaml

```diff
@@ -174,6 +174,8 @@ rules:
   - create
   - delete
   - get
+  - list
+  - watch
```

Comment:
not seeing the need for watch!

Reply:
Not sure why as replied above, I need more investigation.

Reply:
yes I think we need the watch permission. If I remove the watch, the pdb created the first time are not quickly sync'ed in the cache so the controller take a long time to know that there is already a pdb created (so the controller will try a few times to create it and get 'already exists' error in each following reconcile loop).